app/models/payment.rb

```diff
@@ -9,6 +9,8 @@ def self.valid_sources
   end
 
   validates :source, presence: true, inclusion: { in: valid_sources }
-  validates :account, :amount, presence: true
+  validates :account, presence: true
+  validates :amount, presence: true, numericality: true
+  validates :processing_fee, numericality: true, allow_nil: false
```

Comment:
`numericality` doesn't have a `not_equal_to`, but rails 4.X does have an `other_than` (I just checked rails 4.2, might be earlier). Unfortunately, we're still on 3.2, so I used `exclusion: { in: [0] }`, which is a little funky, so I added a #TODO for when we upgrade.